pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -238,7 +238,7 @@
     "eufy" = ps: with ps; [ ]; # missing inputs: lakeside
     "everlights" = ps: with ps; [ pyeverlights ];
     "evohome" = ps: with ps; [ evohome-async ];
-    "ezviz" = ps: with ps; [ ]; # missing inputs: pyezviz
+    "ezviz" = ps: with ps; [ pyezviz ];
```

Comment:
Can we enable new tests in home-assistant?

Reply:
Not now, test will be available for 2021.5.x.